### PR TITLE
Relative links don't work with Shakacode website.

### PR DESCRIPTION
**Pull request by Judahmeek**
## Summary by CodeRabbit

- **Documentation**
	- Updated links in the README.md for better accessibility to migration resources for Postgres and Redis databases.
	- All links now direct to full URLs, ensuring users can easily access the correct documentation regardless of their location in the repository.

### CODE REVIEW

**Review by zzaakiirr — APPROVED** [review]

**Comment by zzaakiirr** [comment]
@Judahmeek We also have relative links to images in `docs/tips.md`, I can't see them in shakacode website, could you fix them too?

**GitHub**
image

**ShakaCode**
Screenshot 2024-12-10 at 10 04 21

**Review by zzaakiirr — CHANGES_REQUESTED** [review]
See comment about relative asset links

File: README.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ Additionally, the documentation includes numerous examples and practical tips fo
 11. [CLI Commands Reference](#cli-commands-reference)
 12. [Mapping of Heroku Commands to `cpflow` and `cpln`](#mapping-of-heroku-commands-to-cpflow-and-cpln)
 13. [Examples](#examples)
-14. [Migrating Postgres Database from Heroku Infrastructure](/docs/postgres.md)
-15. [Migrating Redis Database from Heroku Infrastructure](/docs/redis.md)
-16. [Tips](/docs/tips.md)
+14. [Migrating Postgres Database from Heroku Infrastructure](https://github.com/shakacode/control-plane-flow/docs/postgres.md)
```
Comment (zzaakiirr) [inline README.md:57]: _Here and in similar places_
This link returns `Not Found` response - branch name should be specified
```suggestion
14. [Migrating Postgres Database from Heroku Infrastructure](https://github.com/shakacode/control-plane-flow/tree/main/docs/postgres.md)
```

Comment (justin808) [reply README.md:57]: @Judahmeek We need to figure out how to make relative links work for this repo like they do for https://github.com/shakacode/react_on_rails.

**Comment by Judahmeek** [comment]
@zzaakiirr The image issue is something that will have to be fixed in the sc-website repository.

File: README.md (hunk 1 above)
Comment (Judahmeek) [reply README.md]: @justin808 The ReactOnRails repository has NO relative links.

**Review by zakir-dzhamaliddinov — APPROVED** [review]
It's not related to relative links problem, maybe it's better to open another PR for this fix 🤔 - can we also fix [link on line 482](https://github.com/shakacode/control-plane-flow/pull/246/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R482)?

It should be `https://docs.controlplane.com/reference/workload/types#cron` i/o `https://shakadocs.controlplane.com/reference/workload#cron`

**Review by zzaakiirr — APPROVED** [review]
See https://github.com/shakacode/control-plane-flow/pull/246#pullrequestreview-2510987752

**Pull request merged**